src/energyplus/ForwardTranslator/ForwardTranslateScheduleFixedInterval.cpp

```diff
@@ -58,7 +58,7 @@ namespace openstudio {
 namespace energyplus {
 
   static unsigned startNewDay(IdfObject& idfObject, unsigned fieldIndex, const Date& date) {
-    const std::string s = fmt::format("Through: {:02d}/{:02d}", date.monthOfYear().value(), date.dayOfMonth());
+    const std::string s = fmt::format("Through: {:02d}/{:02d}", month(date.monthOfYear().value()), date.dayOfMonth());
```

Comment:
I think you want this instead

```suggestion
    const std::string s = fmt::format("Through: {:02d}/{:02d}", month(date.monthOfYear()), date.dayOfMonth());
```

* `MontOfYear Date::monthOfyear()`: https://github.com/NREL/OpenStudio/blob/805488732f2779ae9f5c64a18fe3ba50048ed7f5/src/utilities/time/Date.hpp#L285-L286

* month: https://github.com/NREL/OpenStudio/blob/805488732f2779ae9f5c64a18fe3ba50048ed7f5/src/utilities/time/Date.hpp#L127-L128